python/pyarrow/_azurefs.pyx

```
@@ -100,6 +105,9 @@ cdef class AzureFileSystem(FileSystem):
         if account_key:
             options.ConfigureAccountKeyCredential(tobytes(account_key))
             self.account_key = tobytes(account_key)
+        elif sas_token:
+            options.ConfigureSASCredential(tobytes(sas_token))
+            self.sas_token = tobytes(sas_token)
```

Should we prefer `sas_token` to `account_key`?
(SAS token is safer than account key, right?)

---

Then should we use `if sas_token: elif account_key:`?

Or should we raise an error when both of `sas_token` and `account_key` are specified?

---

They should be mutually exclusive. I'll add an error later tonight.

---

This is all set!

---

Could you document it?